tests/codegen/llvm/logical_and_or_different_type.ll

```diff
@@ -5,19 +5,18 @@ target triple = "bpf"
 
 %"struct map_internal_repr_t" = type { ptr, ptr }
 %printf_t = type { i64, %printf_args_t }
-%printf_args_t = type { i64, i64, i64, i64 }
+%printf_args_t = type { i8, i8, i8, i8 }
```

Comment:
Did this work? Or did this code previously generate the runtime error?

Reply:
It worked (and works after this PR as well)